Ruby/Ruby.sublime-syntax

```diff
@@ -540,7 +540,7 @@ contexts:
     - match: =
       scope: keyword.operator.assignment.ruby
       push: after-operator
-    - match: '!+|&&|\|\|'
+    - match: '!+|&&|&\.|\|\|'
```

Comment:
This probably should be part of this match since this isn't a logical operator. It should probably be a separate match with a scope such as `punctuation.accessor.safe.ruby`. The logic here being that `punctuation.accessor.dot.ruby` is used for `.`.

Reply:
>  probably should be part

"shouldn't" ?

Reply:
Yes, you are correct @michaelblyons. I meant to type “shouldn’t”.